apps/roam/src/components/LeftSidebarView.tsx

```diff
@@ -400,12 +572,16 @@ const FavouritesPopover = ({ onloadArgs }: { onloadArgs: OnloadArgs }) => {
 };
 
 const LeftSidebarView = ({ onloadArgs }: { onloadArgs: OnloadArgs }) => {
-  const config = useConfig();
+  const initialConfig = useConfig();
+  const [config, setConfig] = useState(initialConfig);
+  useEffect(() => {
+    setConfig(initialConfig);
+  }, [initialConfig]);
```

Comment:
This `useEffect` should be unncessary because the `useState` is already setting the `initialConfig`